spec/support/vendor_api/vendor_api_spec_helpers.rb

```diff
@@ -13,14 +13,25 @@ def get_api_request(url, options = {})
   end
 
   def post_api_request(url, options = {})
-    post url, { params: { meta: VALID_METADATA } }.merge(auth_headers).deep_merge(options)
+    headers_and_params = { params: { meta: VALID_METADATA } }
+      .deep_merge(auth_headers)
+      .deep_merge(json_header)
+      .deep_merge(options)
+
+    headers_and_params[:params] = headers_and_params[:params].to_json
+
+    post url, headers_and_params
   end
 
   def auth_headers
     unhashed_token = VendorApiToken.create_with_random_token!(provider: currently_authenticated_provider)
     { headers: { 'Authorization' => "Bearer #{unhashed_token}" } }
   end
 
+  def json_header
+    { headers: { 'CONTENT_TYPE' => 'application/json' } }
```

Comment:
Is this more of a constant?

Reply:
:+1: Good idea

Reply:
I turned it into a string literal in the one place it's used